sdk/trafficmanager/Azure.ResourceManager.TrafficManager/src/Generated/RestOperations/EndpointsRestOperations.cs

```diff
@@ -32,11 +33,11 @@ public EndpointsRestOperations(HttpPipeline pipeline, string applicationId, Uri
         {
             _pipeline = pipeline ?? throw new ArgumentNullException(nameof(pipeline));
             _endpoint = endpoint ?? new Uri("https://management.azure.com");
-            _apiVersion = apiVersion ?? "2018-08-01";
+            _apiVersion = apiVersion ?? "2022-04-01-preview";
             _userAgent = new TelemetryDetails(GetType().Assembly, applicationId);
         }
 
-        internal HttpMessage CreateUpdateRequest(string subscriptionId, string resourceGroupName, string profileName, string endpointType, string endpointName, TrafficManagerEndpointData data)
+        internal HttpMessage CreateUpdateRequest(string subscriptionId, string resourceGroupName, string profileName, EndpointType endpointType, string endpointName, TrafficManagerEndpointData data)
```

Comment:
This is a breaking change, can you paste the spec PR that caused this change?

Reply:
Spec PR: https://github.com/Azure/azure-rest-api-specs/pull/21308 

It was there in the earlier version as well https://github.com/Azure/azure-rest-api-specs/blob/main/specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-08-01/trafficmanager.json#L41

Nothing really changed in this version: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/trafficmanager/resource-manager/Microsoft.Network/preview/2022-04-01-preview/trafficmanager.json#L37

Reply:
Removed breaking changes

Reply:
Hi @m-nash, the breaking changes has been removed, could you please take a look?